dictionaries/html/src/html.txt

```diff
@@ -3001,6 +3004,7 @@ vsupne
 vsupnE
 Vvdash
 vzigzag
+wacag
```

Comment:
This is a typo of WCAG, and probably doesn't belong in the HTML dictionary, as mentioned over on https://github.com/streetsidesoftware/cspell-dicts/pull/1933
Unless i'm mistaken on what the word is

Reply:
You are correct, sorry :( I removed it.